### PR TITLE
Vision api

### PR DESCRIPTION
**Pull request by fstamenkovic**
This is a long overdue PR that I wanted to make. It's a small package that I wrote to make it easy to use the Google Vision API from python code. The package can be imported as a module in Python code and my hope is that it'll make calling the Vision API easier and more concise.

### CODE REVIEW

File: google_vision_api/image_analyzer.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+def call_Vision_API(image_path, requested_features):
+    '''
+    Wrapper around the Google Vision API. Enables calling it in one line
+    with image path and requested features.
+    Arguments:
+        - image_path: path to image
+        - requested_features: array of strings with features requested.
+
+    Returns: AnnotateImageResponse
```
Comment (ShafiTek) [inline google_vision_api/image_analyzer.py]: A bit more explanation on what AnnotateImageResponse is would help.

File: google_vision_api/vision_api_demo.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+file_name = os.path.abspath('IMG_0653.jpg')
+
+# Loads the image into memory
+with io.open(file_name, 'rb') as image_file:
+    content = image_file.read()
+
+image = types.Image(content=content)
+
+# Performs label detection on the image file
+response = client.label_detection(image=image)
```
Comment (rmsepic) [inline google_vision_api/vision_api_demo.py]: Does this get brand names at all? Or are the labels only materials and what the item may be?

File: google_vision_api/image_analyzer.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+    request = types.AnnotateImageRequest(image=image, \
+                    features=[label_detection])
+
+    result = client.annotate_image(request)
+
+    if not result.label_annotations:
+        return False
+
+    for label in result.label_annotations:
```
Comment (rmsepic) [inline google_vision_api/image_analyzer.py]: I am getting an error that _reult.label_annotations_ is not iterable.

"line 73, in check_plastic
    for label in res:
TypeError: argument of type 'EntityAnnotation' is not iterable"

**Comment by fstamenkovic** [comment]
Need to re-open

**Pull request merged**